be/src/storage/storage_engine.cpp

```diff
@@ -140,6 +140,27 @@ Status StorageEngine::_open() {
     RETURN_IF_ERROR_WITH_WARN(_update_manager->init(), "init update_manager failed");
 
     auto dirs = get_stores<false>();
+
+    for (auto data_dir : dirs) {
+        uint64_t live_sst_files_size_before = 0;
+        if (!data_dir->get_meta()->get_live_sst_files_size(&live_sst_files_size_before)) {
+            LOG(WARNING) << "data dir " << data_dir->path() << " get_live_sst_files_size failed";
+        }
+        Status s = data_dir->get_meta()->compact();
```

Comment:
Will it slowdown the process restart? Make the crash recover takes a long time?

Reply:
yes;
it will not take too long time, about a few dozen seconds approximately.